actionview/lib/action_view/template_details.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module ActionView
+  class TemplateDetails # :nodoc:
+    class Requested
+      attr_reader :locale, :handlers, :formats, :variants
+
+      def initialize(locale:, handlers:, formats:, variants:)
+        @locale = locale
+        @handlers = handlers
+        @formats = formats
+        @variants = variants
+      end
+    end
+
+    attr_reader :locale, :handler, :format, :variant
+
+    def initialize(locale, handler, format, variant)
```

Comment:
Does this second `initialize` declaration override the one that accepts keyword arguments?

Elsewhere in this changeset, `TemplateDetails.new` is invoked with ordered arguments. Should the keyword argument constructor be removed?

Reply:
these are separate classes

Reply:
Oh wow, I misjudged the indentation! thanks for clearing that up.